src/components/v2v3/shared/FundingCycleConfigurationDrawers/FundingDrawer/FundingForm/DistributionSplitsSection/DistributionSplitsSection.tsx

```diff
@@ -191,18 +192,19 @@ export function DistributionSplitsSection({
                 <Trans>Amounts</Trans>
                 <p className="text-sm font-normal">
                   <Trans>
-                    Distribute a specific amount of funds to entities each
-                    funding cycle. Your distribution limit will equal the{' '}
-                    <strong>sum of all payout amounts.</strong>
+                    Pay out specific amounts of ETH to addresses and projects
+                    each cycle. Any remaining ETH will stay in the project for
+                    future cycles and/or token redemption.
```

Comment:
```suggestion
                    future cycles.
```

feel like that might be fine